doc/api/packages.md

```diff
@@ -689,6 +689,21 @@ exports, while resolving the existing `"node"`, `"node-addons"`, `"default"`,
 
 Any number of custom conditions can be set with repeat flags.
 
+Typical conditions should only contain alphanumerical characters,
+using ":" or "=" as separators if necessary. Anything else may run
+into compability issues outside of node.
+
+In node, conditions can be almost any string, including multi-byte characters and whitespace.
+There's only a few restrictions:
+
+1. They must contain at least one character.
+2. They cannot start with "." since they may appear in places that also
+   allow relative paths.
+3. They cannot contain "," since they may be parsed as a comma-separated
+   list by some CLI tools.
+4. They cannot be numeric property keys like "10" since that can have
+   unexpected effects on property ordering.
```

Comment:
I believe this only applies to integers?

```suggestion
4. They cannot be integer property keys like "10" since that can have
   unexpected effects on object property ordering (see .
```

Reply:
Applied - I assume you meant "property key ordering" and not "key property ordering".